### DISCUSSION TITLE
Switching in song mode from playing arranger mode without restart arrangement

### DISCUSSION THREAD

**Discussion by bukru** [SynthstromAudible/DelugeFirmware#2499] [Ideas]
Hi all,

The idea is to make a compléte song and arrangement, then to start playing in arranger view, then to Switch in song view at a specific part of song to allow jamming and to be able to come back in arranger view to continue playing the song.

Would be awesome isn't it ? : p

**Comment by ok-reza** [comment]
this is already possible, if you start playback in arranger view, at any point you can enter SONG view. you'll notice that the launch pads are dimmed. if you don't touch them, you'll see they automatically mute and unmute based on the arranger. if at any point you launch/mute/solo any clip from SONG view, you'll now just be in full control of the clips again. this is when you would jam. when you want to get back to the arrangement, reenter arranger view and scroll the left most column to where you want playback to continue from. on the right beat, press <>+PLAY and the arranger playback will continue from the left most column

**Reply by bukru** [reply]
what a good new : ) thanks
